src/semver/Version.d

```diff
@@ -0,0 +1,139 @@
+/*******************************************************************************
+
+    Utility to simplify working with semantic versioning entities, including
+    parsing and comparison.
+
+*******************************************************************************/
+
+module semver.Version;
+
+/**
+    Describes SemVer aware version instance
+ **/
+struct Version
+{
+    /// major version number
+    int major;
+
+    /// minor version number
+    int minor;
+
+    /// patch version number
+    int patch;
```

Comment:
I think this is missing some parts of semver: [pre-release](http://semver.org/#spec-item-9) and [build metadata](http://semver.org/#spec-item-10). We'll need it for `+breaking` for example.